README.md

```diff
@@ -40,48 +57,49 @@ The entry in the application's `/server/datasources.json` will look like this:
+| url        | String           | Connection URL of form `mongodb://user:password@host/db`. Overrides other connection settings (see below). |
+| user       | String           | Username to connect to database                                                                            |
+| authSource | String           | Optional. Authentification database name. Usually `"admin"` value.                                         |
 
 If you run a MongoDB with authentification ([Docker's example here](https://github.com/docker-library/docs/tree/master/mongo#mongo_initdb_root_username-mongo_initdb_root_password)), you need to specify which database to authenticate against. More details can be found in [MongoDB documentation on Authentification Methods](https://docs.mongodb.com/manual/core/authentication/#authentication-methods). The default value is usually `"admin"`, like in the official docker image.
 
 **NOTE**: In addition to these properties, you can use additional Single Server Connection parameters supported by [`node-mongodb-native`](http://mongodb.github.io/node-mongodb-native/core/driver/reference/connecting/connection-settings/).
 
 ### Additional properties
 
-- **allowExtendedOperators**: Set to `true` to enable using MongoDB operators such as
-`$currentDate`, `$inc`, `$max`, `$min`, `$mul`, `$rename`, `$setOnInsert`, `$set`, `$unset`, `$addToSet`,
-`$pop`, `$pullAll`, `$pull`, `$pushAll`, `$push`, and `$bit`.  Default is `false`.
-- **enableGeoIndexing**: Set to `true` to enable 2dsphere indexing for model properties
-of type `GeoPoint`. This allows for indexed ```near``` queries.  Default is `false`.
-- **lazyConnect**:
-  - Default is `false`.
-  - If set to `true`, the database instance will not be attached to the datasource and the connection is deferred.
-  - It will try to establish the connection automatically once users hit the endpoint. If the mongodb server is offline, the app will start, however, the endpoints will not work.
-- **disableDefaultSort**: Set to `true` to disable the default sorting
- behavior on `id` column, this will help performance using indexed columns available in mongodb.
-- **collation**: Specify language-specific rules for string comparison, such as rules for lettercase and accent marks. See [`MongdoDB documentation`](https://docs.mongodb.com/manual/reference/collation/) for details. It can also be used to create [`case insensitive indexes`](https://docs.mongodb.com/manual/core/index-case-insensitive/).
+| Property                                                                                        | Type&nbsp;&nbsp; | Default | Description                                                                                                                                                                                                                                                                                                                       |
```

Comment:
nitpick (feel free to ignore): can you make the type column slightly wider to fit `Boolean`?

Reply:
Update table:
<img width="801" alt="Screen Shot 2020-06-03 at 09 44 55" src="https://user-images.githubusercontent.com/50331796/83645215-3746b880-a580-11ea-95de-fa05ee5f9125.png">